src/main/scala/chisel3/util/SRAM.scala

```diff
@@ -104,6 +105,14 @@ class SRAMInterface[T <: Data](
   val writePorts: Vec[MemoryWritePort[T]] = Vec(numWritePorts, new MemoryWritePort(tpe, addrWidth, masked))
   val readwritePorts: Vec[MemoryReadWritePort[T]] =
     Vec(numReadwritePorts, new MemoryReadWritePort(tpe, addrWidth, masked))
+
+  private[chisel3] var _underlying: Option[HasTarget] = None
+
+  /** Set underlying target information. This is intended to store information
+    * about the SRAM that instantiates this interface and should be set to None
+    * by default if this interface was instantiated directly.
+    */
```

Comment:
This comment should be a little more user-facing, something like "Target information for annotating the underlying SRAM if it is known"